src/Voron/Impl/Paging/AbstractPager.cs

```diff
@@ -258,5 +261,28 @@ public static bool IsKeySizeValid(int keySize)
 
             return true;
         }
+
+        public void MaybePrefetchMemory(List<long> pagesToPrefetch)
```

Comment:
* We need to move this to Win32 pager.
* We need to implement this via madvise(WILL_NEED) in Linux